test/AutoRest.Shared.Tests/AutoRest.Shared.Tests.csproj

```diff
@@ -5,7 +5,7 @@
     <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
     <IsPackable>false</IsPackable>
     <Nullable>annotations</Nullable>
-    <NoWarn>SA1649;SA1633</NoWarn>
+    <NoWarn>SA1649;SA1633;NU1605</NoWarn>
```

Comment:
we should share this in the common folder so its only one place to undo later

Reply:
This one can be reverted I think.